Makefile

```diff
@@ -3,11 +3,9 @@ GOLANGCI_LINT_VERSION:=v2.1.6
 .PHONY: mockgen
 mockgen:
 	go install go.uber.org/mock/mockgen@latest
```

Comment:
Unrelated to this change but I wonder if we should target a specific version (at least major version).